Lib/unittest/case.py

```diff
@@ -338,6 +339,25 @@ def __exit__(self, exc_type, exc_value, tb):
                 .format(logging.getLevelName(self.level), self.logger.name))
 
 
+def short_description_from_docstring(text):
```

Comment:
This function name should start with an underscore since it's not meant to be part of the public API. Also, I think a shorter name should be used like `_make_short_description(docstring)` or `_get_short_description(docstring)`. Information about the arguments can be gotten from the argument names or function docstring.